### DISCUSSION TITLE
Original Function Missing?

### DISCUSSION THREAD

**Discussion by xiecsuk** [Dotnet-Boxed/Templates#4] [Q&A]
Before I discovered your excellent updates, I used the original ASP.NET Web Application to start creating a website. It has one function that would make your version so much more useable. Using a large screen to display my first efforts of a web page, I reduced the width of the page to see what happened at the various screen break points. When I reached what I assume was the break point for the phone sizes, the webpage became fluid. Thus the contents of the page reduced in size proportionally to fit the phone width but the formatting of the page remained constant. That removes a lot of extra effort needed to make sure the page looks good on each phone size.

**Comment by RehanSaeed** [comment]
ASP.NET MVC Boilerplate uses <ahref="http://getbootstrap.com/css/#grid">Bootstrap</a> which is built from the ground up to support responsive websites as you describe. You can take a look at the link above to see how this all works. The few sample pages that are generated for you have been tested briefly at all sizes but if you have found a bug, please do let me know.
